tariff/fixed/month.go

```diff
@@ -0,0 +1,133 @@
+package fixed
+
+import (
+	"errors"
+	"fmt"
+	"slices"
+	"strconv"
+	"strings"
+)
+
+//go:generate go tool enumer -type Month
+type Month int
+
+const (
+	January Month = iota
+	February
+	March
+	April
+	May
+	June
+	July
+	August
+	September
+	October
+	November
+	December
+)
+
+var Year = []Month{January, February, March, April, May, June, July, August, September, October, November, December}
+
+var shortMonths = map[string]Month{
+	// english
+	"jan": January,
+	"feb": February,
+	"mar": March,
+	"apr": April,
+	"may": May,
+	"jun": June,
+	"jul": July,
+	"aug": August,
+	"sep": September,
+	"oct": October,
+	"nov": November,
+	"dec": December,
+	// german
+	// "jan": January,
+	// "feb": February,
+	"mär": March,
```

Comment:
I don't think we want to add translations here. If there's no golang OOTB support (and I didn't check), we could allow month numbers in addition to english names. Months empty must equal "any month".

Reply:
The english short names are just the first 3 letters. That leaves the map for the German excepions.

Reply:
@andig was going to do numbers then realised days was done with words. My German is literally non existent.  I think it should be both numbers and words but that is a simple expansion of that map.